src/app.js

```diff
@@ -18,6 +19,11 @@ const eventsRoutes = require('./routes/events');
 
 const app = express();
 const port = process.env.PORT;
+app.use(
```

Comment:
@sarahmood, this will work, I tested it, but it will allow any origin (anyone on the web can use our API) but we want only the front-end app to be able to use our API. 

From the [CORS package](https://www.npmjs.com/package/cors#simple-usage-enable-all-cors-requests), we can do this by replacing this code with the following: 

```
const whitelist = [
    'http://localhost:3000',
    'https://pebble-work.herokuapp.com',
];
const corsOptions = {
    origin: function (origin, callback) {
        if (whitelist.indexOf(origin) !== -1) {
            callback(null, true);
        } else {
            callback(new Error('Not allowed by CORS'));
        }
    },
};
app.use(cors(corsOptions));
```

Reply:
Thank you Nur! I actually doubt that and i asked on slack but i  got no respond! I will change it now.